test/runtime/fqdn.go

```diff
@@ -104,11 +132,18 @@ var _ = Describe("RuntimeFQDNPolicies", func() {
 
 		bindDBCilium     = "db.cilium.test"
 		bindDBOutside    = "db.outside.test"
+		bindDBDNSSSEC    = "db.dnssec.test"
 		bindNamedConf    = "named.conf.local"
 		bindNamedOptions = "named.conf.options"
 
 		world1Target = "http://world1.cilium.test"
 		world2Target = "http://world2.cilium.test"
+
+		DNSSECDomain         = "dnssec.test"
+		DNSSECWorld1Target   = "world1.dnssec.test"
+		DNSSECWorld2Target   = "world2.dnssec.test"
+		DNSSECContainerImage = "docker.io/eloycoto/dnssec-client"
```

Comment:
How did you get the client to accept the keys generated for the test? I guess you can specify the root key? Does the bind instance then return a single response, or do we simulate 2-3 hops with signing? I'm not sure that it matters in the end.

Reply:
discussed offline, made in the docker-entrypoint of the container.